protocol-designer/src/pages/Designer/DeckSetup/LabwareTools.tsx

```diff
@@ -449,27 +451,33 @@ export function LabwareTools(props: LabwareToolsProps): JSX.Element {
           }
         })}
       </Flex>
-      <StyledLabel>
-        <StyledText desktopStyle="bodyDefaultRegular" color={COLORS.grey60}>
-          {t('custom_labware')}
-        </StyledText>
-        <input
-          data-testid="customLabwareInput"
-          type="file"
-          onChange={e => {
-            setSelectedCategory(CUSTOM_CATEGORY)
-            dispatch(createCustomLabwareDef(e))
-          }}
-        />
-      </StyledLabel>
+      <Flex
+        padding={`${SPACING.spacing4} ${SPACING.spacing12}`}
+        alignItems={ALIGN_CENTER}
+        justifyContent={JUSTIFY_CENTER}
+      >
+        <StyledLabel css={BUTTON_LINK_STYLE}>
+          <StyledText desktopStyle="bodyDefaultRegular" color={COLORS.grey60}>
```

Comment:
i think the color on this styled text is preventing the text from having the hover state
```suggestion
          <StyledText desktopStyle="bodyDefaultRegular" >
```